ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -196,6 +196,34 @@ def test_tensorflow_Cosh(
         x=np.asarray(x, dtype=input_dtype),
     )
 
+# Cross
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.Cross"
+    ),
+)
+def test_tensorflow_Cross(
+    dtype_and_x, as_variable, num_positional_args, native_array, fw
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="raw_ops.Cross",
+        x=np.asarray(x, dtype=input_dtype),
```

Comment:
Cross takes 2 arguments so here also there should be 2 inputs, you can refer [here](https://github.com/unifyai/ivy/blob/45d5abc45ecfe815edc4664e20bafb2b2b050c47/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py#L569). I believe this should be able to solve the failing test.

Reply:
Thank you! Fixed :)